docs/reference/camera.md

```diff
@@ -56,6 +58,18 @@ More information on frustums in the corresponding subsection below.
 Aliasing artifacts can appear as jagged edges (or moiré patterns, strobing, etc.).
 Anti-aliasing is a technique that selectively blurs these jagged edges (and thus makes them look more smooth) in the rendered camera image, similar to an anti-aliasing filter in real camera sensors.
 
+- The `ambientOcclusionRadius` field denotes the radius of geometric occlusion searches in the scene.
+Increasing the radius can increase occlusion from further objects.
+However, at lower quality levels, near-field occlusion can start to disappear as the radius increases.
+This effect is disabled by default on cameras for performance reasons.
```

Comment:
We should probably explicitly say that 0 is for disabling.

Reply:
done!